src/main/web/templates/handlebars/list/partials/results-title.handlebars

```diff
@@ -1,3 +1,3 @@
-{{!-- Removed indentation to remove space between title and edition --}}
 <h3 class="search-results__title">
-    <a href="{{uri}}">{{{description.title}}}{{#if_all description.edition showEdition}}{{#if_all (ne description.edition 'yes') (ne description.edition '<strong>yes</strong>')}}: {{{description.edition}}}{{/if_all}}{{/if_all}} {{#if description.latestRelease}}({{labels.latest-release}}){{/if}}</a></h3>
\ No newline at end of file
+        <a href="/redir?url={{uri}}&term={{parameters.query.0}}{{parameters.q.0}}&pageIndex={{result.paginator.currentPage}}&linkIndex={{@index_1}}&type={{listType}}&pageSize={{#if parameters.size.0}}{{parameters.size.0}}{{else}}10{{/if}}">{{{description.title}}}{{#if_all description.edition showEdition}}{{#if_all (ne description.edition 'yes') (ne description.edition '<strong>yes</strong>')}}: {{{description.edition}}}{{/if_all}}{{/if_all}} {{#if description.latestRelease}}({{labels.latest-release}}){{/if}}</a>
```

Comment:
Not sure that '{{parameters.query.0}}' is required for the term field? when debugging I had only a value for '{{parameters.q.0}}'.